src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1215,29 +1234,531 @@ public static VariantContext makeFromAlleles(final String name, final String con
         final int length = alleleStrings.get(0).length();
 
         boolean first = true;
-        for ( final String alleleString : alleleStrings ) {
+        for (final String alleleString : alleleStrings) {
             alleles.add(Allele.create(alleleString, first));
             first = false;
         }
-        return new VariantContextBuilder(name, contig, start, start+length-1, alleles).make();
+        return new VariantContextBuilder(name, contig, start, start + length - 1, alleles).make();
+    }
+
+    /**
+     * Split variant context into its biallelic components if there are more than 2 alleles
```

Comment:
Alright, so this method existed in GATK3 and was used by `LeftAlignAndTrimAlleles`. In GATK4 however most of the functionality of this cluster of methods already exists in the utils suite. Specifically, if you look at the class `AlleleSubsettingUtils` most of the work this class is doing in reconstructing things like the genotype annotations are already done by that class. Indeed it would probably be best to reduce code duplication to simply expand that class and call into it from this method rather than have two different codepaths subsetting alleles in much the same way.